indico_toolkit/indico_wrapper/modelop.py

```diff
@@ -0,0 +1,204 @@
+from indico import IndicoClient
+from indico.queries import GraphQLRequest
+import json
+import string
+
+
+class ModelOp:
+
+    """
+    Class to assit in Model Group Operation calls.
+    """
+
+    def __init__(self, client: IndicoClient):
+        self.client = client
+        self.task_type = None
+
+    def get_model_group(self, model_group_id: int) -> dict: 
+        """
+        Get specified model group information.
+        Args:
+            model_group_id (int): id of model group 
+        Returns:
+            dict: attributes of the specified model group id 
+        """
+
+        query = """
+        
+query GetModelGroup($id: Int) {
+  modelGroups(modelGroupIds: [$id]) {
+    modelGroups {
+      id
+      name
+      status
+      taskType
+      selectedModel {
+        id
+        status
+      }
+      modelOptions {
+        id
+        samplingStrategy
+        modelTrainingOptions
+      }
+    }
+  }
+}
+                """
+        
+        results = self.client.call(
+            GraphQLRequest(query, {"modelGroupId": model_group_id})
+        )
+        models = results["modelGroups"]["modelGroups"]
+        for model in models:
+            if model["id"] == model_group_id:
+                #assign task type for advanced model options
+                self.task_type = model["taskType"]
+                #assign model settings for validation
+                self.model_setings = json.loads(model["modelOptions"]["modelTrainingOptions"])
+                return model
+
+    def retrain_model(self, model_group_id: int):
+        """
+        Forces model retrain, used within update_model_settings to apply new Training Options.
```

Comment:
This doesn't have to be used with update_model_settings, I think this just runs a model retrain (whether you've updated training options or not)-- update doc string